Public/JS/pg.js

```diff
@@ -0,0 +1,9 @@
+ let formname=document.querySelector("#form_name");
+ let email = document.querySelector("#form_email");
+ let message= document.querySelector("#message_form");
+
+if(formname==""){console.log("Ingresa el campo texto!");}
+else if (email == ""){console.log("Ingresa un email!");}
+else if(message=""){
+     console.log("Ingresa un mensaje!");
```

Comment:
Codacy has a fix for the issue: [Delete `···`](https://app.codacy.com/gh/sanchezabad034/CALIDADSOFTWARE/pullRequest?prid=9675051)
```suggestion
  console.log("Ingresa un mensaje!");
```